packages/common/src/csv.ts

```diff
@@ -0,0 +1,15 @@
+import { isNotFalsy } from './is-not-falsy';
+
+/**
+ * Converts a (simple) CSV string into a cleaned list.
+ *
+ * Note that this generic is an unsafe type cast.
+ */
+export const csv = <T extends string = string>(
```

Comment:
Don't love just calling this `csv` when it also allows for other delimiters, but I do like the simplicity of just `csv` too and anything else won't be as clean and easy 🤔

Reply:
I like this. I went with it.

Reply:
I think that would be better. 

My original thought was `delimiterSeparatedValue` but I think that's really wordy and also you might have to explain what a delimiter is. 

cleanSplit
stringSplit
csvStringSplit 

Oh the time we spend as devs on naming 😜